docs/source/markdown/podman-down-kube.1.md

```diff
@@ -0,0 +1,14 @@
+% podman-down-kube(1)
+
+## NAME
+podman-down-kube - Stop containers, pods or volumes based on Kubernetes YAML
+
+## SYNOPSIS
+**podman down kube** [*options*] *file.yml|-*
+
+## DESCRIPTION
+**podman down kube** will tear down the pods created by a previous run of `podman play kube`. It reads in a structured file of Kubernetes YAML.  It will then stop the containers, pods or volumes described in the YAML. If the yaml file is specified as "-" then `podman down kube` will read the YAML file from stdin.
```

Comment:
```suggestion
**podman down kube** will tear down the pods created by a previous `podman play kube` run. It reads in a structured file of Kubernetes YAML. It will then stop the containers, pods, or volumes described in the YAML. If the YAML file is specified as "-" then `podman down kube` will read the YAML file from stdin.
```